src/test/java/fr/techad/edc/httpd/search/GetAndDeployServiceTest.java

```diff
@@ -0,0 +1,174 @@
+package fr.techad.edc.httpd.search;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+
+import org.apache.commons.io.FileUtils;
+import org.apache.commons.io.filefilter.TrueFileFilter;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import com.networknt.config.Config;
+
+import fr.techad.edc.httpd.UploadHandler;
+import fr.techad.edc.httpd.WebServerConfig;
+import fr.techad.edc.httpd.utils.TokenUtils;
+import io.undertow.server.HttpServerExchange;
+import io.undertow.util.HttpString;
+
+public class GetAndDeployServiceTest {
+  static final String CONFIG_NAME = "webserver";
+  static WebServerConfig config = (WebServerConfig) Config.getInstance().getJsonObjectConfig(CONFIG_NAME,
+      WebServerConfig.class);
+  static GetAndDeployService service = new GetAndDeployService(config);
```

Comment:
To avoid to declare some field as static to be initalized in BeforeAll method, you can annotate the class with : 
**@TestInstance(TestInstance.Lifecycle.PER_CLASS)**